### ISSUE TITLE
 Create collapsible section UI and apply to Permit and SIWE Signatures

### ISSUE THREAD

**Issue by digiwand** [MetaMask/metamask-extension#24680]
### What is this about?

Create a collapsible section UI and apply to Permit (Raw Message) and SIWE

This may affect scrollToBottom logic

Blocked by: https://github.com/MetaMask/metamask-extension/issues/24617 (SIWE)

### Design

Permit: [Figma](https://www.figma.com/design/wcXUl6AH5KNFwKdAIv49kh/Confirmation-redesign-V5?node-id=3478-5929&t=UMQoESYrbRrEZJ2R-0)
SIWE: [Figma](https://www.figma.com/design/v8npDTFD2fofLjDfbJUD9j/Confirmation-redesign-V3?node-id=360-4572&t=7yeUu97HSM2fWHVf-0)

### Stakeholder review needed before the work gets merged

- [ ] Engineering (needed in most cases)
- [ ] Design
- [ ] Product
- [ ] QA (automation tests are required to pass before merging PRs but not all changes are covered by automation tests - please review if QA is needed beyond automation tests)
- [ ] Security
- [ ] Legal
- [ ] Marketing
- [ ] Management (please specify)
- [ ] Other (please specify)

**Comment by bschorchit** [comment]
Closing this for now as this will be replaced by a new issue for collapsing the message section in the UI
